packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -496,23 +511,23 @@ class TextPainter {
   ///
   /// Valid only after [layout] has been called.
   double get minIntrinsicWidth {
-    assert(!_needsLayout);
```

Comment:
Maybe slightly more readable to define a getter `bool get _needsLayout => _paragraph != null` and use it here and elsewhere?

Reply:
`_paragraph` is declared nullable, so even with NNBD these could be null at this point and that would be a bug.